cmd/generate.go

```diff
@@ -42,10 +57,25 @@ func initConfig() {
 
 	log.Debug("config path:", defaultConfigFilePath)
 	if err := config.LoadConfig(defaultConfigFilePath); err != nil {
-		log.Debug("error while loading the configuration file. Loading the defaults")
+		log.Fatalf("error while loading the configuration file: %s. Configuration file must be present.", defaultConfigFilePath)
 	}
 }
 
 func Execute() error {
 	return generateCmd.Execute()
 }
+
+// ParseOverrides checks for command line flags for the overrides and updates the viper global object
+func ParseOverrides(g generator.Generator, cmd *cobra.Command) {
+	m, err := cmd.Flags().GetStringToString("overrides") // check for --overrides flag for map[string]string
+	if err != nil {
+		log.Warn("No overrides defined. Continuing with defaults defined in the config file.")
+		return
+	}
+	vNew := viper.New() // new viper object for storing overrides
+	for k, v := range m {
+		vNew.Set(k, v)
+	}
+
+	viper.MergeConfigMap(vNew.AllSettings()) // merge to global viper object
```

Comment:
the return value (error) should be checked

Reply:
Done